docs/antctl.md

```diff
@@ -151,6 +151,22 @@ specified local Pod using this `antctl` command:
 antctl get networkpolicy -p pod -n namespace
 ```
 
+#### Mapping endpoints to NetworkPolicies
```

Comment:
Just to confirm you use 4 "#" by purpose to make it a sub-section of "NetworkPolicy commands" and not list it in the table of contents?

Reply:
Yes it was on purpose. I believe given that we support a single query command right now and it's tied to NPs, this is the right place to put this section.